src/main/java/com/post/menagment/controller/PostOfficeController.java

```diff
@@ -0,0 +1,34 @@
+package com.post.menagment.controller;
+
+import com.post.menagment.model.PostOffice;
+import com.post.menagment.services.PostOfficeService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class PostOfficeController {
+
+    @Autowired
+    private PostOfficeService postOfficeService;
+
+
+    @RequestMapping(value = "/create", method = RequestMethod.POST)
+    public HttpStatus addNewPostOffice(@RequestBody PostOffice postOffice) {
+        postOfficeService.create(postOffice);
+        return HttpStatus.OK;
```

Comment:
It's better to use HttpStatus.CREATED (201) in such case.

Reply:
done